### ISSUE TITLE
Question client n°5

### ISSUE THREAD

**Issue by aureliencoste** [2023-2024-ECOM-INFO5-G01/docs#3]
Comment le dispositif (qui permet d'identifier la quantité de chaque aliment consommé) remonte les informations ?

**Comment by SybilleCaffiau** [comment]
sous quelle forme vous voulez dire ? idéalement le poids consommé (estimé consommé) de chaque plat et c'est l'application qui calcule la quantité de calories que cela représente

**Comment by guerinlo** [comment]
Les informations (poids consommés) sont remontés directement à l'application qui calcul automatiquement la quantité de calories que cela représente ? 
Est ce qu'une fois que le calcul est effectué, les soignants et/ou medecin on besoin de remplir à la main les informations dans le dossier patient ou est ce que le logiciel rempli automatiquement l'information ?

**Comment by SybilleCaffiau** [comment]
il faudrait effectivement prévoir le calcul des calories à partir du poids consommé de chaque plat
Vous n'avez pas à faire le lien avec le dossier patient (c'est hors de votre application)

**Comment by guerinlo** [comment]
Que fait on de la donnée calculé si elle n'est pas ajoutée au dossier patient ?

**Comment by SybilleCaffiau** [comment]
Dossier patient est un terme existant dans ce domaine : [ https://industriels.esante.gouv.fr/segur-du-numerique-en-sante/segur-vague-1/se-faire-referencer/le-segur-du-numerique-pour-l-hopital/le-dossier-patient-informatise-dpi-du-systeme-d | https://industriels.esante.gouv.fr/segur-du-numerique-en-sante/segur-vague-1/se-faire-referencer/le-segur-du-numerique-pour-l-hopital/le-dossier-patient-informatise-dpi-du-systeme-d ] 

est ce cela d’auquel vous faites référence ? 


De: "guerinlo" ***@***.***> 
À: "2022-2023-ECOM-INFO5-G06" ***@***.***> 
Cc: "SybilleCaffiau" ***@***.***>, "Assign" ***@***.***> 
Envoyé: Mardi 10 Octobre 2023 14:24:28 
Objet: Re: [2023-2024-ECOM-INFO5-G01/docs] Question client n°5 (Issue #3) 





Que fait on de la donnée calculé si elle n'est pas ajoutée au dossier patient ?

**Comment by guerinlo** [comment]
Oui, je fais référence au dossier dans lequel on stock toutes les informations pour le patient

**Comment by SybilleCaffiau** [comment]
Dans ce cas, je maintient que vous n avez pas accès à ce dossier. Vos données sont indépendantes du dossier patient informatisé 


De: "guerinlo" ***@***.***> 
À: "2022-2023-ECOM-INFO5-G06" ***@***.***> 
Cc: "SybilleCaffiau" ***@***.***>, "Assign" ***@***.***> 
Envoyé: Mardi 10 Octobre 2023 16:57:55 
Objet: Re: [2023-2024-ECOM-INFO5-G01/docs] Question client n°5 (Issue #3) 





Oui, je fais référence au dossier dans lequel on stock toutes les informations pour la patient

**Comment by guerinlo** [comment]
Je parle du dossier patient interne à notre application, créée par le.a médecin/soignant.e dans lequel il.elle rentre les informations sur le poids, l'age ....

**Comment by SybilleCaffiau** [comment]
I faut que les données soient automatiquement associées aux autres données patient de votre système
